test/test_nn.py

```diff
@@ -2353,7 +2353,7 @@ def test_threshold_int(self):
 
     def test_embedding_sparse_basic(self):
         embedding = nn.Embedding(10, 20, sparse=True)
-        input = Variable(torch.LongTensor([[0, 2, 4, 5], [4, 3, 0, 9]]))
+        input = torch.LongTensor([[0, 2, 4, 5], [4, 3, 0, 9]])
```

Comment:
we should try to use new-style constructors. All of the `torch.{Type}Tensor` constructors are deprecated. The replacement for `LongTensor` is `torch.tensor(data, dtype=torch.long)`